configs/platforms/huaweios-6-ppc.rb

```diff
@@ -0,0 +1,11 @@
+platform "huaweios-6-ppc" do |plat|
+  plat.servicedir "/lib/systemd/system"
+  plat.defaultdir "/etc/default"
+  plat.servicetype "systemd"
+  plat.codename "jessie"
+
+  plat.add_build_repository "http://pl-build-tools.delivery.puppetlabs.net/debian/pl-build-tools-release-#{plat.get_codename}.deb"
+  plat.provision_with "export DEBIAN_FRONTEND=noninteractive; apt-get update -qq; apt-get install -qy --no-install-recommends build-essential devscripts make quilt pkg-config debhelper rsync fakeroot"
+  plat.install_build_dependencies_with "DEBIAN_FRONTEND=noninteractive; apt-get install -qy --no-install-recommends "
+  plat.vmpooler_template "debian-8-x86_64"
```

Comment:
Can we have a comment somewhere in here that explains the intention to cross compile? Otherwise it looks weird to see ppc and then x86_64 in the definitions.